README.md

```diff
@@ -37,19 +37,21 @@ not the actual file-system path).
 ##### dotfiles
 
 Set how "dotfiles" are treated when encountered. A dotfile is a file
-or directory that begins with a dot ("."). Note this check is done on
-the path itself without checking if the path actually exists on the
-disk. If `root` is specified, only the dotfiles above the root are
-checked (i.e. the root itself can be within a dotfile when when set
-to "deny").
+or directory that begins with a dot (".").
 
+Note: This check is only performed on the path (of the url).
+It doesn't affect `root` (of the filesystem) and doesn't actually check for dotfiles on disk.
```

Comment:
Cool. Once we get this into a position we feel good about, since this is a later large change with formatting as well, I usually ask one of my documentation writer co-workers to look it over. I may have more feedback in the coming weeks based on her feedback :)